README.md

```diff
@@ -1,25 +1,49 @@
 # Spdx-Java-Library
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.spdx/java-spdx-library/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.spdx/java-spdx-library)
+[![javadoc](https://javadoc.io/badge2/org.spdx/java-spdx-library/javadoc.svg)](https://javadoc.io/doc/org.spdx/java-spdx-library)
 ![Java CI with Maven](https://github.com/spdx/Spdx-Java-Library/workflows/Java%20CI%20with%20Maven/badge.svg)
 
-Java library which implements the Java object model for SPDX and provides useful helper functions.
-
-The API documentation is available at:
-<https://spdx.github.io/Spdx-Java-Library/>
-
-## Code quality badges
-
 [![Bugs](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=bugs)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 [![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=security_rating)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 [![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=java-spdx-library&metric=sqale_index)](https://sonarcloud.io/dashboard?id=java-spdx-library)
 
+Spdx-Java-Library is a Java library which implements the Java object model for SPDX and provides useful helper functions.
+
+The library is available in Maven Central as
+[`org.spdx:java-spdx-library`](https://search.maven.org/artifact/org.spdx/java-spdx-library)
+(note the order of the word "java-spdx").
+
+If you are using Maven, you can add the following dependency in your POM file:
+
+```xml
+<dependency>
+  <groupId>org.spdx</groupId>
+  <artifactId>java-spdx-library</artifactId>
+  <version>(,2.0]</version>
```

Comment:
I believe this version number range should be `[2.0,3.0)` i.e. "greater than or equal to 2.0, but less than 3.0".

`(,2.0]` means "less than or equal to 2.0", which won't work given that 1.x of the library and 2.x of the library are not mutually compatible.

Reply:
Updated.